src/RemoteHttpInterceptor.ts

```diff
@@ -118,17 +116,19 @@ export class RemoteHttpResolver extends Interceptor<HttpRequestEventMap> {
         return
       }
 
-      const isomorphicRequest: IsomorphicRequest = JSON.parse(
-        serializedRequest,
-        requestReviver
-      )
+      const requestJson = JSON.parse(serializedRequest, requestReviver)
+      log('parsed intercepted request', requestJson)
 
-      log('parsed intercepted request', isomorphicRequest)
+      const body = requestJson.body ? bufferFrom(requestJson.body) : undefined
 
-      const interactiveIsomorphicRequest: InteractiveIsomorphicRequest = {
-        ...isomorphicRequest,
-        respondWith: createLazyCallback(),
-      }
+      const isomorphicRequest = new IsomorphicRequest(requestJson.url, {
+        ...requestJson,
+        body: body?.buffer || new ArrayBuffer(0),
```

Comment:
`new ArrayBuffer(0)` is the default iso request body value if `init.body` is undefined. Perhaps we could remove this pipe `||`?